sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraServiceImpl.java

```diff
@@ -95,7 +93,7 @@ public boolean start() throws IOException {
     }
 
     @Override
-    public boolean advance() throws IOException {
+    public boolean advance() {
```

Comment:
How come we no longer have the `IOException` here and elsewhere?

Reply:
The static analyzer of IntelliJ complains for this and apparently is valid to override a method and don't throw exceptions (if the method does not: JLS https://docs.oracle.com/javase/specs/jls/se8/html/jls-11.html#jls-11.2 second paragraph (almost impossible to understand given the not so clear redaction).